src/main/java/com/powsybl/openloadflow/util/ContingencyContext.java

```diff
@@ -0,0 +1,69 @@
+package com.powsybl.openloadflow.util;
+
+import com.powsybl.contingency.Contingency;
+import com.powsybl.contingency.ContingencyElement;
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Switch;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.openloadflow.sa.BranchTripping;
```

Comment:
@floriand-e2r I imagine that the Branchtripping class has to move too.

Reply:
Yes sure! util -> sa dependency is forbidden